src/config/packets.h

```diff
@@ -7,18 +7,21 @@
  * rAthena configuration file (http://rathena.org)
  * For detailed guidance on these check http://rathena.org/wiki/SRC/config/
  **/
-
+//2008_09_10a
 #ifndef PACKETVER
-	#define PACKETVER 20151104
+	#define PACKETVER 20141022 //20131223 //20101116//20100730//20151104
 #endif
 
+#define NO_PACKET_OBFUSCATION
+
```

Comment:
this entire file looks weird. it is completely different to what exists inside the master branch. merging this into master would change the file entirely.